.azure-pipelines/main.yml

```diff
@@ -5,3 +5,4 @@ pr:
 
 jobs:
   - template: templates/jobs/standard-tests-jobs.yml
+  - template: templates/jobs/documentation-tests-jobs.yml
```

Comment:
I think you can merge the doc test job in the `standard-tests-jobs.yml`, since it is not required to separate it from the standard test suite.